scripts/jenkins/ardana/heat-ses-cluster.yaml

```diff
@@ -0,0 +1,338 @@
+    type: OS::Heat::SoftwareConfig
+    properties:
+      config:
+        str_replace:
+          params:
+            $sles_pool: { get_param: sles_pool_repo }
+            $sles_updates: { get_param: sles_updates_repo }
+            $ses_pool: { get_param: ses_pool_repo }
+            $ses_updates: { get_param: ses_updates_repo }
+          template: |
+            #!/bin/bash
+            set -e
+            zypper --non-interactive --quiet --gpg-auto-import-keys ar -f $sles_pool SLES-Pool
+            zypper --non-interactive --quiet --gpg-auto-import-keys ar -f $sles_updates SLES-Updates
+            zypper --non-interactive --quiet --gpg-auto-import-keys ar -f $ses_pool SES-Pool
+            zypper --non-interactive --quiet --gpg-auto-import-keys ar -f $ses_updates SES-Updates
+
+  ses_config_salt_master:
+    type: OS::Heat::SoftwareConfig
+    properties:
+      config: |
+        #!/bin/bash
+        set -e
+        # ntp
+        zypper -n in -l -y ntp
+        cat <<EOF > /etc/ntp.conf
+        server ntp1.suse.de
+        server ntp2.suse.de
+        server ntp3.suse.de
+        EOF
+        systemctl enable ntpd.service
+        systemctl start ntpd.service
+
+        # salt-minion
+        zypper -n in -l -y salt-minion hwinfo iptables
+        cat <<EOF > /etc/salt/minion.d/master.conf
+        master: localhost
+        EOF
+        systemctl enable salt-minion.service
+        systemctl start salt-minion.service
+
+        # salt-master
+        zypper -n in -l -y salt-master
+        cat <<EOF > /etc/salt/master.d/auto_accept.conf
+        auto_accept: True
+        EOF
+
+        systemctl enable salt-master.service
+        systemctl start salt-master.service
+
+  ses_config_salt_minion:
+    type: OS::Heat::SoftwareConfig
+    properties:
+      config:
+        str_replace:
+          params:
+            $salt_master_ip: { get_attr: [ses-admin, networks, { get_resource: ses_network }, 0]} 
+          template: |
+            #!/bin/bash
+            set -e
+            # ntp
+            zypper -n in -l -y ntp
+            cat <<EOF > /etc/ntp.conf
+            server ntp1.suse.de
+            server ntp2.suse.de
+            server ntp3.suse.de
+            EOF
+            /usr/bin/systemctl enable --now ntpd.service
+
+            # salt-minion
+            zypper -n in -l -y salt-minion hwinfo iptables
+            cat <<EOF > /etc/salt/minion.d/master.conf
+            master: $salt_master_ip
+            EOF
+            systemctl enable salt-minion.service
+            systemctl start salt-minion.service
+
+  ses_config_deepsea:
+    type: OS::Heat::SoftwareConfig
+    properties:
+      config: |
+        #!/bin/bash
+        set -e
+        # wait a bit so the minions can connect
+        sleep 2m
+        salt-key --list-all
+        master_minion=$(hostname -f)
+        zypper -n in -l -y deepsea
+        cat <<EOF > /srv/pillar/ceph/stack/global.yml
+        stage_prep_master: default-no-update-no-reboot
```

Comment:
no update means what? we might want to install SES with updates ..

Reply:
given that the update repos are there, we don't have to update during stage0. See https://www.suse.com/documentation/suse-enterprise-storage-5/singlehtml/book_storage_admin/book_storage_admin.html#deepsea.rolling_updates